client-side-js/allControls.js

```diff
@@ -1,6 +1,6 @@
-async function clientSide_allControls(controlSelector) {
+async function clientSide_allControls(controlSelector, browserInstance) {
     controlSelector = await Promise.resolve(controlSelector) // to plug into fluent async api
-    return await browser.executeAsync((controlSelector, done) => {
```

Comment:
Why no use of `browserInstance` in files `client-side-js/` `_navTo, getUI5Version, injectTools, injectUI5, testLibrary`?